db-init-action/action.yml

```diff
@@ -7,19 +7,66 @@ inputs:
   db-name:
     description: 'Database name'
     required: true
+  schema-name:
+    description: 'Schema name'
+    required: true
   gcp-project-id:
     description: "GCP project of the database"
     required: true
   cloudsql-superuser:
     description: "User created by Terraform in CloudSQL"
     required: true
-  db-init-script-path:
-    description: "Path to init script relative to repository root"
+  db-user-for-app:
+    description: "User created by this workflow. The user will be used by the service"
     required: true
 
 runs:
   using: "composite"
   steps:
+    # Get DB User for application Secret from secret manager
+    - name: Get Username from Secret Manager
+      id: get-db-username-for-app
+      uses: 'google-github-actions/get-secretmanager-secrets@v1'
+      with:
+        secrets: |-
+          token:${{ inputs.gcp-project-id }}/${{ inputs.db-user-for-app }}-username
+      continue-on-error: true
+
+    - name: Get Password from Secret Manager
+      id: get-db-user-password-for-app
+      uses: 'google-github-actions/get-secretmanager-secrets@v1'
+      with:
+        secrets: |-
+          token:${{ inputs.gcp-project-id }}/${{ inputs.db-user-for-app }}-password
+      continue-on-error: true
+    
+    - name: Set up gcloud cli
+      uses: 'google-github-actions/setup-gcloud@v1'
+      if: steps.get-db-username-for-app.outcome != 'success' || steps.get-db-user-password-for-app.outcome != 'success'
+
+    - name: Save username to a file
+      shell: bash
+      run: | 
+        echo ${{ inputs.db-user-for-app }} > db-username.txt
```

Comment:
use `echo "DB_USER=${{ inputs.db-user-for-app }}" >> $GITHUB_ENV` so we don't have to cleanup more

Reply:
ok, you should be able to pipe it ala
`printf "s3cr3t" | gcloud secrets create my-secret --data-file=-`